components/hdmi_cec/hdmi_cec.cpp

```diff
@@ -251,7 +252,7 @@ bool HDMICEC::send(uint8_t source, uint8_t destination, const std::vector<uint8_
 }
 
 SendResult IRAM_ATTR HDMICEC::send_frame_(const std::vector<uint8_t> &frame, bool is_broadcast) {
-  InterruptLock interrupt_lock;
+  pin_->detach_interrupt();  // do NOT listen for pin changes while sending
```

Comment:
Something I missed in the initial review: still on an ESP32 ESP-IDF build, this change (with the corresponding `attach_interrupt` at the end) seems to make WiFi unstable. When I change only this part back to using `InterruptLock`, the issue goes away. Did this change improved something on ESP8266 in your testing?

Probably related to the issue @DIYtechie mentioned recently on #10.